src/test/java/org/cactoos/io/UncheckedBytesTest.java

```diff
@@ -50,7 +50,7 @@ public void rethrowsCheckedToUncheckedException() {
     @Test
     public void worksNormallyWhenNoExceptionIsThrown() throws Exception {
         final Text source = new TextOf("hello, cactoos!");
-        MatcherAssert.assertThat(
+        new Assertion<>(
```

Comment:
@marceloamadeu you forgot to add `affirm()`

Reply:
@victornoel Done!